generators/app/__snapshots__/generator.spec.ts.snap

```diff
@@ -255,6 +259,22 @@ exports[`generator - app with default config should match snapshot 1`] = `
   "camelizedBaseName": "jhipster",
   "capitalizedBaseName": "Jhipster",
   "caret": undefined,
+  "ciCd": undefined,
+  "ciCdAny": false,
+  "ciCdAzure": false,
+  "ciCdCircle": false,
+  "ciCdGithub": false,
+  "ciCdGitlab": false,
+  "ciCdIntegrations": undefined,
+  "ciCdIntegrationsAny": false,
+  "ciCdIntegrationsCypressDashboard": false,
+  "ciCdIntegrationsDeploy": false,
+  "ciCdIntegrationsHeroku": false,
+  "ciCdIntegrationsPublishDocker": false,
+  "ciCdIntegrationsSnyk": false,
+  "ciCdIntegrationsSonar": false,
+  "ciCdJenkins": false,
+  "ciCdTravis": false,
```

Comment:
Is Travis still being used by developers?

Reply:
I use it personally: the only CI provider that offer Vt-x machines (for nested virt). Still, it was here before, I just changed the code approach to fit with current best practices